cachito/workers/pkg_managers/yarn.py

```diff
@@ -69,7 +69,22 @@ def get_yarn_proxy_repo_username(request_id):
     return f"cachito-yarn-{request_id}"
 
 
-def _get_deps(yarn_lock, file_deps_allowlist):
+def find_non_dev(non_dev_deps, yarn_lock, current_dep):
```

Comment:
It misses docstring and annotations.

I don't actually understand what this function is doing, to me, it seems it only creates a list of all dependencies, where is that `non_dev` or `find` logic?

Reply:
Fixed!